### DISCUSSION TITLE
I Get the Following Error When Performing Make -j 6 in the modules folder of MOOSE.

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#28934] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

"MooseApp.C:1132:42: warning: ignoring returned value of type 'PetscErrorCode', declared with attribute 'nodiscard' [-Wunused-result]  1132 |     Moose::PetscSupport::petscSetupOutput(_command_line.get()); "

**Comment by GiudGiud** [comment]
Hello

This is just a warning. 
We are fixing it in a coming PR. Thanks for reporting

Guillaume
